docs/Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 	find $(SOURCEDIR) -name "*.rst" | grep -v index | xargs rm -f
 
 apidoc:
-	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/visualization ../datasetinsights/dashboard.py ../datasetinsights/data/datasets/protos
+	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/ststs/visualization ../datasetinsights/dashboard.py ../datasetinsights/datasets/protos
 html: apidoc
 	sphinx-build -b html $(SOURCEDIR) $(BUILDDIR)/html
 	@echo
```

Comment:
When I track this branch, and do `make html`, why do I get this error:
```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/sphinx/config.py", line 319, in eval_config_file
    execfile_(filename, namespace)
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 88, in execfile_
    exec(code, _globals)
  File "/Users/bowen.li/Documents/dataset-insights/docs/source/conf.py", line 21, in <module>
    release = pkg_resources.get_distribution(project).version
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 482, in get_distribution
    dist = get_provider(dist)
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 358, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 901, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/bowen.li/miniconda3/envs/bowen/lib/python3.7/site-packages/pkg_resources/__init__.py", line 787, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'datasetinsights' distribution was not found and is required by the application

make: *** [html] Error 2
```

Reply:
You need to run `poetry install` to install datasetinsights.

Reply:
Got it.